root - WARNING - This message will get logged on to a file
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1, 1).
tensorflow - WARNING - Model was constructed with shape (None, 1, 1662) for input KerasTensor(type_spec=TensorSpec(shape=(None, 1, 1662), dtype=tf.float32, name='lstm_31_input'), name='lstm_31_input', description="created by layer 'lstm_31_input'"), but it was called on an input with incompatible shape (None, 1, 1).
